ebs_snapshots/file_backup_config.py

```diff
@@ -0,0 +1,45 @@
+from backup_config import BackupConfig
+import yaml
+import jsonschema
+import kayvee
+
+schema = {
+    "type": "object",
+    "name": "Volume backup schedule",
+    "properties": {
+            "max_snapshots": {"type": "number"},
+            "interval": {"type": "string"},
+    },
+    "additionalProperties": False,
+}
+
+class FileBackupConfig(BackupConfig):
+
+    config = {}
+    path = ""
+
+    def __init__(self, path):
+        self.path = path
+
+    def refresh(self):
+        """ Read file and return parsed yml """
+        with open(self.path, "r") as open_file:
+            return yaml.load(open_file)
+
+    @classmethod
+    def _validate_config(cls, new_config):
```

Comment:
wondering if `_validate_config` needs to be part of `BackupConfig` since should be shared across all implementations